monitoring-and-tracing/script.adoc

```diff
@@ -4,21 +4,24 @@
 
 == Introduction
 
-TODO
+In the Cloud-Native world monitoring and distributed tracing are the crucial parts of the service not visible to the naked eye, but vital for DevOps in order to obtain various system metrics, easily investigate underline issues and identify potential performance bottlenecks. When the cloud IDE is deployed on a cluster it is utterly important to have enough observability in order to make sure that the development environment is in a good shape, stable and ready to be used. 
```

Comment:
Some language suggestions:

* "underline issues" should be "underlying issues"
* I would suggest "critical" instead of "utterly important"
* I think you can remove "in order" from "in order to" without changing the sentence meaning
* Things are "in good shape" not "in a good shape"